Okay, thanks for the pointers. `python310.tests` fails like this.

```
==================================== ERRORS ====================================
__________________ ERROR collecting ast3/tests/test_basics.py __________________
ImportError while importing test module '/build/source/ast3/tests/test_basics.py'.
Hint: make sure your test modules/packages have valid Python names.
Traceback:
/nix/store/hwaq99xsvx89y7p25njjq6nq2zm0qh9v-python3-3.10.0a5/lib/python3.10/importlib/__init__.py:126: in import_module
    return _bootstrap._gcd_import(name[level:], package, level)
ast3/tests/test_basics.py:5: in <module>
    from typed_ast import _ast3
E   ImportError: /nix/store/pz8kk94jvxdz8y73qzx2pxi94p6f0rbf-python3.10-typed-ast-1.4.3/lib/python3.10/site-packages/typed_ast/_ast3.cpython-310-x86_64-linux-gnu.so: undefined symbol: _PyArena_AddPyObject
=========================== short test summary info ============================
ERROR ast3/tests/test_basics.py
!!!!!!!!!!!!!!!!!!!! Interrupted: 1 error during collection !!!!!!!!!!!!!!!!!!!!
=============================== 1 error in 0.16s ===============================
```

We wanted to do a python-unstable run, would it be appropriate to retarget to python-unstable, or should it go to staging?

--- a/pkgs/development/interpreters/python/default.nix
+++ b/pkgs/development/interpreters/python/default.nix
@@ -181,10 +181,10 @@ in {
     sourceVersion = {
       major = "3";
       minor = "9";
-      patch = "4";
+      patch = "5";
       suffix = "";
     };
-    sha256 = "Sw5mRKdvjfhkriSsUApRu/aL0Jj2oXPifTthzcqaoTQ=";
+    sha256 = "10vdf46q5ldnzkprm8pldvr5a9hrdpxjv7mpzgdw6vj3cl318nhc";
     inherit (darwin) configd;
     inherit passthruFun;
   };

--- a/pkgs/development/interpreters/python/default.nix
+++ b/pkgs/development/interpreters/python/default.nix
@@ -122,10 +122,10 @@ with pkgs;
       sourceVersion = {
         major = "3";
         minor = "8";
-        patch = "9";
+        patch = "10";
         suffix = "";
       };
-      sha256 = "XjkfPsRdopVEGcqwvq79i+OIlepc4zV3w+wUlAxLlXI=";
+      sha256 = "1n8rjb3jn0j8dvi1qn94rxayc9rh982d8wgkrjy41n1x15k4mwka";
     };
   };
 

--- a/pkgs/development/interpreters/python/default.nix
+++ b/pkgs/development/interpreters/python/default.nix
@@ -195,9 +195,9 @@ in {
       major = "3";
       minor = "10";
       patch = "0";
-      suffix = "a5";
+      suffix = "b3";
     };
-    sha256 = "BBjlfnA24hnx5rYwOyHnEfZM/Q/dsIlNjxnzev/8XU0=";
+    sha256 = "05fc4mp2ysb372bzkwbn1b1z01bfldnaqig6rxmif58hs3aawrr2";
     inherit (darwin) configd;
     inherit passthruFun;
   };

